XXXX-fee-market-for-storage.md

```diff
@@ -0,0 +1,121 @@
+---
+simd: 'XXXX'
```

Comment:
As per the PR number, the SIMD value here should be `0008` and the filename updated to `0008-fee-market-for-storage.md`